src/lib/db.js

```diff
@@ -64,6 +90,8 @@ function executeScript (filename) {
 
   if (dbType === 'strong-oracle') {
     return executeSQLPlus(filepath)
+  } else if (dbType === 'pg') {
+    return executePSQL(filepath)
```

Comment:
There are scripts in the SQL file that don't work when run line by line: https://github.com/interledger/five-bells-ledger/blob/master/src/sql/pg/create.sql#L191